tests/test_iqm_provider.py

```diff
@@ -31,3 +33,10 @@ def test_get_backend(linear_architecture_3q):
     assert backend.client._base_url == url
     assert backend.num_qubits == 3
     assert set(backend.coupling_map.get_edges()) == {(0, 1), (1, 2)}
+
+
+def test_client_signature():
+    url = 'http://some_url'
+    provider = IQMProvider(url)
+    backend = provider.get_backend()
+    assert backend.client._signature == f'iqm-client {version("iqm-client")}, qiskit-iqm {version("qiskit-iqm")}'
```

Comment:
From the `IQMProvider` test perspective it makes sense to check if it passes the provided `client_signature` further, leaving exact `_signature` format for `iqm-client` to decide:
```suggestion
    assert f'qiskit-iqm {version("qiskit-iqm")}' in backend.client._signature
```

Reply:
fixed